prototype-kit/app/views/confirmation.html

```diff
@@ -0,0 +1,31 @@
+{% extends "layouts/main.html" %}
+
+{% block pageTitle %}
+Confirmation page template – {{ serviceName }} – GOV.UK Prototype Kit
+{% endblock %}
+
+{% block content %}
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    {{ govukPanel({
```

Comment:
As mentioned in my other comment, the final step of this process will be downloading the CSV file(s) so we should probably rephrase / remake some of this page. Doesn't need to be anything special maybe just "Click here to download your files"